### PR TITLE
Reorganize the Get Started files, revise the Get Started introduction, and fix a handful of issues.

### PR DESCRIPTION
**Pull request by samglover**
This is a start at overhauling the Get Started pages. It also fixes a handful of small issues that aren't necessarily related.

The big change is the intro page (now at /docs/get_started). I rewrote some of the copy, removed outdated information or put it in context, and reorganized the page. I also added examples showing the Document Assembly Line in action, links to our one-pagers, and information about joining the Document Assembly Line community and partnering with the LIT Lab.

On the backend, I consolidated the Get Started files in /docs/get_started, fixed the links broken by that change, and fixed a bunch of broken anchors I found when building the site. (I ignored broken anchors related to ALKiln pages since @plocket is still working on those pages.)

### CODE REVIEW

File: docs/contributors.mdx [hunk 1]
```diff
@@ -244,4 +244,4 @@ The following organizations and groups have generously donated the time of their
 * <a href="https://www.appgeo.com/"  >AppGeo</a>
 * <a href="https://www.bentley.edu/academics/graduate-programs/masters-human-factors"  >Bentley University Court UX Team</a>
 * <a href="https://community.lawyer/"  >Community Lawyer</a>
-* <a href="https://www.documate.org/"  >Documate</a>
+* <a href="https://www.documate.org/"  >Documate</a>
```
Comment (nonprofittechy) [inline docs/contributors.mdx:247]:
```suggestion
* <a href="https://gavel.io/"  >Gavel (formerly known as Documate)</a>
```

File: docs/contributors.mdx [hunk 2]
```diff
@@ -244,4 +244,4 @@ The following organizations and groups have generously donated the time of their
 * <a href="https://www.appgeo.com/"  >AppGeo</a>
 * <a href="https://www.bentley.edu/academics/graduate-programs/masters-human-factors"  >Bentley University Court UX Team</a>
 * <a href="https://community.lawyer/"  >Community Lawyer</a>
```
Comment (nonprofittechy) [inline docs/contributors.mdx:246]:
```suggestion
* <a href="https://afterpattern.com//"  >AfterPattern (formerly known as Community Lawyer)</a>
```

**Review by nonprofittechy — APPROVED** [review]
Looks good! Two thoughts:

1. Make sure there are redirects for all of the moved pages. I think you added some, so I'm guessing the answer is yes.
2. Any files under /reference are auto generated, so it's not that useful to edit them in this repo. We need to edit the docstring in the relevant package.

**Comment by samglover** [comment]
> 1. Make sure there are redirects for all of the moved pages. I think you added some, so I'm guessing the answer is yes.

I definitely broke links, but I fixed everything I found by building the repo.

> 2. Any files under /reference are auto generated, so it's not that useful to edit them in this repo. We need to edit the docstring in the relevant package.

Good to know!

**Comment by samglover** [comment]
> 1. Make sure there are redirects for all of the moved pages. I think you added some, so I'm guessing the answer is yes.

I was thinking about fixing broken links, not redirects. Different things!

Just added redirects for all the moved pages.

**Pull request merged**